components/usage/pkg/db/cost_center.go

```diff
@@ -180,6 +184,7 @@ func (c *CostCenterManager) UpdateCostCenter(ctx context.Context, newCC CostCent
 				return CostCenter{}, err
 			}
 
+			newCC.BillingCycleStart = NewVarcharTime(now)
```

Comment:
Seems a little assymmetrical to have the start time but not the end time. 
Do we know the end time for stripe as well?

Reply:
@svenefftinge Good catch. We know the (approximate) end time for Stripe -- it's the last millisecond of the calendar month.

However, I wasn't fully sure of the implications of setting it. Will Gitpod do anything when it reaches a `nextBillingTime` for a Stripe CostCenter? (This would be problematic, because it's in fact Stripe's responsibility to issue and finalize an invoice, so I think Gitpod shouldn't do anything proactively on the Cost Center and instead wait for Stripe's callback.)

Reply:
Yes, we wouldn't do the invoice finalization, of course. But this is clearly communicated through the `billingStrategy`. I just thought it seems strange that we have a starting time but no end time. One could argue that the start time is managed by stripe as well.